### DISCUSSION TITLE
Failed to resolve versions

### DISCUSSION THREAD

**Discussion by radin100** [Cyfrin/security-and-auditing-full-course-s23#130] [Q&A] [answered]
When I try to pull forge build on rio-network new sherlock audit contest I got this output:
```shell
Encountered invalid solc version in contracts/interfaces/IRioLRT.sol: No solc version exists that matches the version requirement: =0.8.23
Encountered invalid solc version in contracts/interfaces/IRioLRTAVSRegistry.sol: No solc version exists that matches the version requirement: =0.8.23
Encountered invalid solc version in contracts/interfaces/IRioLRTAssetRegistry.sol: No solc version exists that matches the version requirement: =0.8.23
Encountered invalid solc version in contracts/interfaces/IRioLRTCoordinator.sol: No solc version exists that matches the version requirement: =0.8.23
...
...
...
Encountered invalid solc version in test/utils/beacon-chain/MockBeaconChainOracle.sol: No solc version exists that matches the version requirement: =0.8.23
```

I tried to change the solc version, by installing and set 0.8.23 but still got this output

**Comment by tobezhanabi** [comment]
Trying asking them there at their discord

**Reply by radin100** [reply]
Other users don't have this Error. This Error comes from 'unexisting' version required. But I do not know how to make it exist

**Comment by alymurtazamemon** [comment, marked as answer]
@radin100 First of you should run `foundryup` command so can get up to date version and second try setting this solc version to default profile.

```
[profile.default]
  solc = "0.8.23"
```

Here you can find more information [link](https://github.com/foundry-rs/foundry/tree/58a272997516046fd745f4b3c37f91d0eb113358/config#all-options)

**Reply by radin100** [reply]
much love
